internal/controller/util_test.go

```diff
@@ -98,7 +98,7 @@ func NewPVCAndPV(
 	}
 	return pvc, pv
 }
-func WaitForFinBackupIsReady(ctx context.Context, finbackup *finv1.FinBackup) {
+func MakeFilBackupReady(ctx context.Context, finbackup *finv1.FinBackup) {
```

Comment:
Thank you. However, there are typos that should be fixed.
```suggestion
func MakeFinBackupReady(ctx context.Context, finbackup *finv1.FinBackup) {
```